src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/StandardsController.cs

```diff
@@ -55,6 +56,11 @@ public async Task<IActionResult> ViewStandards()
                 standard.ConfirmRegulatedStandardUrl = standard.IsApprovalPending ? Url.RouteUrl(RouteNames.GetConfirmRegulatedStandard, new { Ukprn, standard.LarsCode }) : string.Empty;
             }
 
+            if(Request.GetTypedHeaders().Referer.ToString() != null && Request.GetTypedHeaders().Referer.ToString().Contains("delete-standard"))
```

Comment:
Use tempdata instead of referrer. On post delete, set a variable in the tempdata and check that variable here.

Reply:
@DevenKShah It appears to throw an Error 500 to the api, with the message, being something like:
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException

Reply:
this is now done